.github/workflows/build.yaml

```diff
@@ -22,9 +22,7 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
```

Comment:
Was there any value in `fetch-depth: 0`?

Reply:
you're right `make tag` will not work with shallow clone. We should fix this as there is no sense in pulling the whole history